### DISCUSSION TITLE
Deleted codespace

### DISCUSSION THREAD

**Discussion by MidhiaQueiroz** [community/community#68311] [Codespaces] [answered]
### Select Topic Area

Question

### Body

 I committed my work as usual but failed to push my changes up to my branch. Because our codespaces error out when restarted, I deleted the codespace I was working on and started a new one this morning.  Is there a way to recover those commits or recover that deleted codespace?

**Comment by mehulrajdev** [comment, marked as answer]
For those of you looking for this same issue, please open a support ticket with git hub. They can manually restore your deleted code space but give them some time to resolve. Hopefully, they will be able to roll something out that will enable us to restore codespaces previously deleted ourselves.
